src/provider/Provider.ts

```diff
@@ -573,23 +571,21 @@ export class Provider {
     }
   }
 
-  /** Get status for a specific jobId/documentId/owner.
-   * @param {string} jobId
-   * @param {number} index
-   * @param {string} providerUri
-   * @param {string} destination
-   * @param {Web3} web3
-   * @param {AbortSignal} signal abort signal
+  /** Get compute result url
+   * @param {string} providerUri The URI of the provider we want to query
+   * @param {Web3} web3 Web3 instance
+   * @param {string} consumerAddress The consumer ethereum address
+   * @param {string} jobId The ID of a compute job.
+   * @param {number} index Result index
    * @return {Promise<ComputeJob | ComputeJob[]>}
```

Comment:
return type here is different than TypeScript-typed return type `Promise<string>`